README.md

```diff
@@ -0,0 +1,126 @@
+# spark-rewards
+
+This implements revision D of [Meridian: Off-chain scheduled rewards](https://spacemeridian.notion.site/Meridian-Off-chain-scheduled-rewards-f9480ef009464ecfaf02a4f752cc4d70).
+
+## Routes
+
+### `GET /scheduled-rewards`
+
+Response:
+
+```js
+{
+  "address1": "scheduledRewards1",
+  "address2": "scheduledRewards2",
+  // ...
+}
+```
+
+### `GET /scheduled-rewards/:address`
+
+Response:
+
+```js
+"scheduledRewards1"
+```
+
+### `POST /scores`
+
+Request:
+
+```js
+{
+  "participants": {
+    "address1": "score1",
+    "address2": "score2"
+  },
```

Comment:
In `spark-evaluate`, we have this data stored as an object:

https://github.com/filecoin-station/spark-evaluate/blob/ae885a9dcac95a9c3265634511eae5779e7370a5/lib/evaluate.js#L148

Therefore it's easiest to submit it as this. Splitting it up using `Object.keys()` and `Object.values()` is easy too, so that shouldn't be a problem. But, storing it as two variables is only because of the limitation of Solidity - I believe service interfaces shouldn't be influenced by this limitation. It also doesn't really make this service simpler.

Reply:
Thinking about it a bit more, your other comment about the ordering of object keys, and how it's hacky to rely on it, convinced me to change this to accept arrays instead. Will follow up!

Reply:
Done!